rest/models/owasp/suppressions.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) Connexta
+  ~
+  ~ This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License
+  ~ as published by the Free Software Foundation, either version 3 of the License, or any later version.
+  ~
+  ~ This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+  ~ See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+  ~ <http://www.gnu.org/licenses/lgpl.html>.
+  -->
+<!--
+    Contains the suppressions that should not be inherited by dependent projects.
+-->
+<suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.2.xsd">
+    <suppress>
+        <!--
+            OWASP infers that jackson-databind-nullable 0.1.0 is related to Jackson 0.1.0
+            dependencies and flags vulnerabilities on cpe:2.3:a:fasterxml:jackson:0.1.0 and
+            cpe:2.3:a:fasterxml:jackson-databind:0.1.0, which do not exist.
+            GitHub issue https://github.com/jeremylong/DependencyCheck/issues/2047 was opened to
+            get this vulnerability resolved in OWASP/DependencyCheck itself. This suppression should
+            be removed when that issue has been resolved.
```

Comment:
😆 You just made my day with this great comment. Thanks

Reply:
❓ Should we start adding a history with each suppression? Like the date it was added and the above comment. That way, when it needs to be revisited later by a developer, they can add to it indicating it is still ok and we will be able to track down how often a given suppression has been re-analyzed and by whom using git blame (assuming nobody else touches that particular line)